integrationtesting/opendcs-tests/src/test/resources/data/Comps/ToIrregular/Test1/timeseries/expectedOutputs/test1-inst-output.tsimport

```diff
@@ -1,13 +1,13 @@
 ﻿SET:TZ=UTC
-SET:UNITS=in
-TSID:testsite-pattern.Precip-Cum.Inst.~1Hour.0.output
```

Comment:
i would add this location and keep the well define name. easier to distinguish issues if any,